Standardize pathways

LienReyserhove opened the pull request:
I updated the vocabulary of `pathway` in the dataset as suggested in the [updated vocabulary](https://github.com/trias-project/vocab/blob/master/vocabulary/pathway.tsv).

I'm not 100% sure whether this suggested vocabulary covers all terms, e.g.:

|pathway                   |suggested vocabulary |
|:-------------------------|:--------------------|
|aquaculture               |escape_aquaculture   |
|mariculture |escape_aquaculture   |
|Aquaculture / mariculture |escape_aquaculture   |

Now, there's no distinction between aquaculture and mariculture in the suggested vocabulary, while they do make the distinction in the [CBD standard](https://www.cbd.int/doc/meetings/sbstta/sbstta-18/official/sbstta-18-09-add1-en.pdf). For me, this is a loss of information... Or am I wrong?

LienReyserhove committed:
Standardize pathways

see [vocabulary]( https://github.com/trias-project/vocab/blob/master/vocabulary/pathway.tsv)

LienReyserhove committed:
Merge branch 'master' into Pathway_mapping

LienReyserhove committed:
Undo conflicts with base branch

peterdesmet reviewed (changes requested):
1. The CBD vocab does *not* make a distinction between aquaculture and mariculture: it's listed in one term. Although this is a slight loss of information, we'll stick to the vocab rather than inventing our own terms
2. `stowaway_transport` should be `stowaway_other`
3. Can you add a bit more text before the mapping, at least referencing the vocab we're using (https://github.com/trias-project/vocab/tree/master/vocabulary/pathway)?
4. Prepend codes with `cbd_2014_pathway:`

Thanks!

LienReyserhove committed:
Integrate changes by @peterdesmet

- Change `stowaway_transport to `stowaway_other`
- add prefix `cbd_2014`
- Provide more information on the mapping

LienReyserhove committed:
Correct error (:)

LienReyserhove committed:
Generate description.csv and dwc_checklist.md

LienReyserhove committed:
Integrate comment by @peterdesmet

Comment: maybe use a paste() to add cbd_2014_pathway to the non-empty values. Seems more robust.
